en/pages/bank-account.md

```diff
@@ -2,33 +2,43 @@
 
 If you plan to work or live in Berlin, having a German bank account becomes mandatory. Without one you will not be able to receive your salary, rent an apartment or get a postpaid mobile phone. Fortunately, opening a bank account is usually simple and fast.
 
-Below you will find the steps required to open a Deutsche Bank (DB) and N26 bank account, but keep in mind that there are many other banks available. Choosing a bank is a very personal matter, so that decision is at your discretion.
+Below you will find the steps required to open a bank account. 
+
+## Normal Banks
+
+These banks have physical branches where you may visit and open you bank account. One can also open bank account without going to a branch by using VideoIdent / PostIdent processes offered by banks. However, this option is not available to people from certain nations like India ( [Reddit.com](https://www.reddit.com/r/germany/comments/6v66sz/whats_up_with_postident_not_accepting_foreign/) ).
```

Comment:
Small typo, it should be "your" instead of "you".

I would remove the second sentence ("One can also open a bank account...") or add an example of a traditional bank that allow customers to open an account without visiting one of the branches.

Reply:
Yeah, its best suited in Online banks section. I think it should be more clear there.